### ISSUE TITLE
Black rectangle in EP6 Tips

### ISSUE THREAD

**Issue by FatesTimelines1337** [07th-mod/umineko-answer#63]
On the Tips screen for EP6, executing the Erika profile on the second page of Characters results in a black rectangle being displayed.
![20190711114058_1](https://user-images.githubusercontent.com/38626341/61065116-06422000-a3d1-11e9-907e-4cb8a0ad9a71.jpg)

EDIT: Alright, the profile is meant to be blank but that black box is not meant to be there

**Comment by drojf** [comment]
My initial investigation shows that this is due to the file `Umineko Chiru\big\bmp\r_click\cha_tati\ep6_2\eri_1d.png`. The file (in the ps3 version) uses indexed mode (as opposed to RGB or Greyscale).

The image is supposed to be a large transparent rectangle

I did some testing and it appears the game doesn't properly support indexed mode:
- only half the image is rendered
- if the image's first pixel is greyscale, greyscale colors are treated as transparency
- any actual transparent pixels are rendered as black
- ...probably other wierd things as well

I know the engine treats certain images (.bmp files) differently (looks at their first pixel), but for .png files normally it ignores this. It appears that using indexed mode makes the engine turn on "special mode" when looking at the images, or perhaps it's just not supported correctly.

This probably happened when we optimized all the .pngs a while ago, and some images were converted to indexed mode.

To fix this issue, it would probably be a good idea to convert any images which are in indexed mode to RGB32/RGB32A mode. I'm not sure if there's an easy way to do this besides writing a script to detect any images in indexed mode, then converting them (like if imageMagick can do it in a one-liner).

The broken image is attached below:

[eri_1d broken](https://user-images.githubusercontent.com/1249449/61104266-fef64180-a4b8-11e9-94c9-7b951f3e6c48.png)

**Comment by drojf** [comment]
I did a scan for indexed images and these are the images which are indexed. However I think most of these images display correctly, and only the ones with transparency don't display correctly (although I'm not sure why the image in Umineko Chiru\big\bmp\r_click\cha_tati\ep6_2\eri_1d.png appears as a plain red image. 

Note that this list might include files which are not part of our mod, as I just scanned the whole game directory (only takes a few secs to do the scan).

[answer_arcs_affected_files.txt](https://github.com/07th-mod/umineko-answer/files/3385944/answer_arcs_indexed_pixel_files.txt)

**Comment by drojf** [comment]
I did some more digging. It turns out that the images are only interpreted in the wrong "mode" if you specify the `:ba;` prefix instead of the usual `:b;` prefix. 

The 'a' part states 

> a - for 'alphablend'; the right half of the image contains alpha-level data for the left half  

The reason this was not broken before, is that the game will ignore the `a` if you are using a non-indexed image. For indexed images, it respects the `a` and does this funky stuff.

The `:ba;`  (and `:ab;` etc.) does come up in the script quite a few times, I'll need to have a look whether it's feasible to just fix up all those cases. Any time a .bmp file is used, we need to keep the `a` part since that's the only way to have transparent .bmp images.

**Comment by drojf** [comment]
I think this issue and all related issues should be fixed now.

I decided it would be safer to fix the images which are in indexed mode (asset only fix). All new installations should have this issue fixed (and possibly an other cases it happens in the game, but i'm not sure how many there are).

The asset change in the installer was made here: https://github.com/07th-mod/python-patcher/commit/134bca07e22c726307a33c34292b923bfd7e1f88 . Manual download links in the wiki were also updated here: https://github.com/07th-mod/wiki/commit/66cc79adeeaaf37674764f5f40b2b88fc3b2c12c

Umineko graphics archive  v2 -> v3
Umineko Answer graphics archive v1->v2
Umineko Answer rolling update all OS was also updated
